service/history/historyEngine.go

```diff
@@ -211,81 +213,138 @@ func (e *historyEngineImpl) StartWorkflowExecution(startRequest *h.StartWorkflow
 	serializedHistory, serializedError := msBuilder.hBuilder.Serialize()
 	if serializedError != nil {
 		logging.LogHistorySerializationErrorEvent(e.logger, serializedError, fmt.Sprintf(
-			"HistoryEventBatch serialization error on start workflow.  WorkflowID: %v, RunID: %v", executionID, runID))
+			"HistoryEventBatch serialization error on start workflow.  WorkflowID: %v, RunID: %v",
+			*execution.WorkflowId, *execution.RunId))
```

Comment:
Can you use generated getters for WorkflowId and RunId

Reply:
fixed